snap/snapcraft.yaml

```diff
@@ -575,6 +586,26 @@ parts:
       craftctl default
       snap-helpers write-hooks
 
+  libvirt-exporter:
+    plugin: go
+    source-tag: 2.3.3
+    source-type: git
+    source: https://github.com/Tinkoff/libvirt-exporter
+    build-snaps:
+      - go
+    after:
+      - libvirt
+    build-packages:
+      - pkg-config
+      - libvirt-dev
```

Comment:
As we build libvirt from source in a previous part can we use the built artefacts from that step rather than using libvirt-dev - I'm concerned we'll end up with a conflict of some sort between the libvirt package components and the built versions.

Reply:
Thanks for the comment. Indeed, that might be a concern. 

I've looked into the source of `libvirt`, and I found that the source should already contain all the necessary files and shared library to build the libvirt exporter. I updated my patch to build the exporter with appropriate `CGO_*` flags.